third_party/nvfuser/csrc/ir_base_nodes.h

```diff
@@ -361,6 +365,37 @@ class TORCH_CUDA_CU_API Val : public Statement {
 
   void resolveIndexDtype();
 
+  //! Get vector of log messages for this object
```

Comment:
nit: I'd prefer to have the type explicitly stated as `std::vector<...>` than having a comment saying it's a vector.